packages/types/src/index.ts

```diff
@@ -11,3 +11,4 @@ export * from './schema';
 export * from './stat';
 export * from './summary';
 export * from './timing';
+export * from './util';
```

Comment:
[need help] Where is the best place to define the type of ComputedAccessibleNode.

Reply:
I want to avoid util because its role is unclear.
I think `src / browser.ts` is valid.

Reply:
`src/browser.ts` it sounds good and is clear type role.